test/integration/targets/delegate_to/test_delegate_to.yml

```diff
@@ -1,9 +1,11 @@
 - hosts: testhost3
```

Comment:
Oh, no that was before I was running it in containers. The container based tests have the necessary loopback interfaces, so we shouldn't need to be any preflight checks to see if the loopback IPs are up.